### DISCUSSION TITLE
Disable highlighting in minimap

### DISCUSSION THREAD

**Discussion by jkwphysics** [streetsidesoftware/vscode-spell-checker#699] [Q&A]
Is there any way to just have the squiggles in the current view of the editor, but not gum up the minimap scrollbar with those highlights? Most code has tons of variables that are false positives (negatives?).

**Comment by Jason3S** [comment]
You are not the only one asking for this: #504 #272 

That would be a great suggestion for the VS Code team.

This extension uses the built in [Diagnostic API](https://code.visualstudio.com/api/references/vscode-api#Diagnostic) from the [Visual Studio Code Extension API](https://code.visualstudio.com/api/references/vscode-api) to report issues.

Because this extension uses the built in API, it has no control over the display of the issues.
